Gemfile.lock

```diff
@@ -306,3 +306,6 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
+
```

Comment:
This PR should remove the `activerecord_json_validator` gem because this PR renders it obsolete.